### PR TITLE
Updated CodePlex link & text

### PR DESCRIPTION
**Pull request by doddag**
# Replace/remove CodePlex links

Replaced the CodePlex text with GitHub
Repalced the CodePlex link with GitHub\Microsoft

as this link https://github.com/Microsoft seems appropriate

## Summary

Updated the CodePlex links/text

## Details

Replaced the CodePlex text with GitHub
Repalced the CodePlex link with GitHub\Microsoft

as this link https://github.com/Microsoft seems appropriate

## Suggested Reviewers

@mairaw

### CODE REVIEW

**Comment by mairaw** [comment]
Related to issue #2974

**Comment by mairaw** [comment]
@doddag I went through the PR and I'm slightly confused by the changes on the second commit. There are a lot of code files being added that are not related to the CodePlex change. Perhaps the easier thing to do is to remove the last commit and continue with the changes to remove CodePlex from our docs. Let me know if you need any help or directions on how to accomplish that. I can also do that on your behalf if you gave maintainers permissions.

My expectation for this PR would be to only touch the files returned by the search I've included in the issue: https://github.com/dotnet/docs/search?utf8=%E2%9C%93&q=codeplex&type=

Thanks so much for working on this issue!

**Comment by mairaw** [comment]
Closed and reopened to trigger a new CLA validation.

File: docs/framework/windows-workflow-foundation/samples/index.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ These samples provide instruction on various aspects of [!INCLUDE[wf](../../../.
...
   
 > [!NOTE]
->  These samples were created with [!INCLUDE[vs2010](../../../../includes/vs2010-md.md)] and [!INCLUDE[netfx40_short](../../../../includes/netfx40-short-md.md)], but are compatible with [!INCLUDE[vs_current_long](../../../../includes/vs-current-long-md.md)] and [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)]. Additional samples for [!INCLUDE[wf2](../../../../includes/wf2-md.md)] in [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)] can be found on [Codeplex](http://aka.ms/WF45Samples).  
+>  These samples were created with [!INCLUDE[vs2010](../../../../includes/vs2010-md.md)] and [!INCLUDE[netfx40_short](../../../../includes/netfx40-short-md.md)], but are compatible with [!INCLUDE[vs_current_long](../../../../includes/vs-current-long-md.md)] and [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)]. Additional samples for [!INCLUDE[wf2](../../../../includes/wf2-md.md)] in [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)] can be found on [Microsoft Docs](https://docs.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/samples/).
```
[the remark this answers is not in the document]
Comment (doddag) [reply docs/framework/windows-workflow-foundation/samples/index.md]: I have updated the aka.ms to https://code.msdn.microsoft.com/site/search?f%5B0%5D.Type=Technology&f%5B0%5D.Value=Windows%20Workflow%204.5

Will do a commit shortly

thank you

**Comment by doddag** [comment]
Hi I have reverted my 2 nd commit
and updated the Work flow foundation code samples link  to https://code.msdn.microsoft.com/site/search?f%5B0%5D.Type=Technology&f%5B0%5D.Value=Windows%20Workflow%204.5

File: docs/framework/windows-workflow-foundation/samples/index.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ These samples provide instruction on various aspects of [!INCLUDE[wf](../../../.
...
->  These samples were created with [!INCLUDE[vs2010](../../../../includes/vs2010-md.md)] and [!INCLUDE[netfx40_short](../../../../includes/netfx40-short-md.md)], but are compatible with [!INCLUDE[vs_current_long](../../../../includes/vs-current-long-md.md)] and [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)]. Additional samples for [!INCLUDE[wf2](../../../../includes/wf2-md.md)] in [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)] can be found on [Codeplex](http://aka.ms/WF45Samples).  
+>  These samples were created with [!INCLUDE[vs2010](../../../../includes/vs2010-md.md)] and [!INCLUDE[netfx40_short](../../../../includes/netfx40-short-md.md)], but are compatible with [!INCLUDE[vs_current_long](../../../../includes/vs-current-long-md.md)] and [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)]. Additional samples for [!INCLUDE[wf2](../../../../includes/wf2-md.md)] in [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)] can be found on [Microsoft Code](https://code.msdn.microsoft.com/site/search?f%5B0%5D.Type=Technology&f%5B0%5D.Value=Windows%20Workflow%204.5)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/samples/index.md]: Microsoft Code -> Code MSDN

no need to update the link. I'll get someone to update the aka.ms link

File: docs/csharp/getting-started/additional-resources.md [hunk 3]
```diff
@@ -45,8 +45,8 @@ The following sites can help you find answers to common problems.
  [Microsoft Help and Support](http://go.microsoft.com/fwlink/?LinkID=108287)  
  Provides access to Knowledge Base (KB) articles, downloads and updates, support Webcasts, and other services.  
   
- [CodePlex](http://go.microsoft.com/fwlink/?LinkId=137330)  
- Hosts open source software projects. You can use CodePlex to create new projects to share with the world or join others who have already started their own projects.  
+ [Microsoft](https://github.com/Microsoft)
```
[the remark this answers is not in the document]
Comment (mairaw) [reply docs/csharp/getting-started/additional-resources.md]: I agree with you @svick. Wondering if we should just pull this link out from here or even kill this page on a separate PR. @BillWagner?

File: docs/framework/windows-workflow-foundation/samples/index.md (hunk 2 above)
Comment (mairaw) [reply docs/framework/windows-workflow-foundation/samples/index.md]: We have updated the aka.ms link so we just need to update the link text here

File: docs/csharp/getting-started/additional-resources.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (BillWagner) [reply docs/csharp/getting-started/additional-resources.md]: I think we should kill this page. It really doesn't serve its original purpose anymore.

**Review by mairaw — APPROVED** [review]

**Pull request merged**